pages/EXP-Collector.md

```diff
@@ -0,0 +1,7 @@
+The EXP-Collector is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to collect exp.
```

Comment:
Should also mention how, like the fact that it stores it in Flasks of Knowledge.